main.js

```diff
@@ -79,6 +79,44 @@ var obstacleSpike = function(x, y) {
   ctx.fillStyle = "black";
   ctx.fill();
 };
+/*I am making a program on github
+and this is my game hero !!! */
+
+
+var y = 10;
+
+/*body
+and color*/
+
+ctx.fill(105, 73, 75);
+ctx.ellipse(y + 190, 350, 93, 93);
+
+/*big
+smile*/
+
+ctx.fill(245, 240, 240);
+ctx.arc(y + 192, 350, 71, 77, 1, 180);
+
+/*eyes and
+pupils*/
+
+ctx.ellipse(y + 179, 332, 24, 24);
+ctx.ellipse(y + 200, 332, 24, 24);
+ctx.fill(0, 0, 255);
+ctx.ellipse(y + 179, 332, 10, 10);
```

Comment:
Remember that the first argument is the `x` coordinate. Of course you can name your variable anything you like, but in order to avoid confusion when you read the code later you might want to use an `x` here instead :wink: